### PR TITLE
🐛 Cannot share study and 🎨 Various style fixes 

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

Updated - 198/12/2023

Fixed the focus state of password on login page and general input fields update. See profile page

Updated - 18/12/2023
Aligned the product colors with the action buttons and fixed the color alignment mention by @SCA-ZMT and @mguidon in the chat.

- Issue fix - Cannot share services

- various color and style alignments and fixes

before
image2

after
Screenshot 2023-12-14 at 23 44 44
Screenshot 2023-12-14 at 23 45 08

- make the dark mode a little brighter

Screenshot 2023-12-14 at 23 40 10
Screenshot 2023-12-14 at 23 40 38
Screenshot 2023-12-14 at 23 40 54
Screenshot 2023-12-14 at 23 41 02
Screenshot 2023-12-14 at 23 41 21

## Related issue/s

- fixes Cannot share services #5152

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by elisabettai — APPROVED** [review]
Looks good, thanks! 
In the first animation I don't see that you fixed the problem with the Services (e.g. we cannot see with whom they're shared), but I trust you that it has been fixed. 😉

**Comment by jsaq007** [comment]
> Looks good, thanks! In the first animation I don't see that you fixed the problem with the Services (e.g. we cannot see with whom they're shared), but I trust you that it has been fixed. 😉

I saw there was something else I missed over the weekend so I fixed it too as only one side of the fix was done. @mguidon I thinks this should resolve the issue. Perhaps you could build it locally to confirm?

Screenshot 2023-12-17 at 01 14 14

File: services/static-webserver/client/source/class/osparc/theme/Appearance.js [hunk 1]
```diff
@@ -251,6 +268,46 @@ qx.Theme.define("osparc.theme.Appearance", {
       }
     },
 
+    "pb-study/lock-status":  {
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/theme/Appearance.js:271]: About this. Makes no sense to me to inherit the appearance of its father component. But what do I know about QLoookxL

**Review by ignapas — APPROVED** [review]
Sorry I cannot test everything, changing the database to the correct state is just too time consuming

**Review by sanderegg — APPROVED** [review]
looking very nice!

**Review by matusdrobuliak66 — COMMENTED** [review]
:+1:

**Review by elisabettai — APPROVED** [review]
It looks very nice, thanks a lot!

I might have some comments/questions on the colors, but we can discuss offline at some point. 😉

File: services/static-webserver/client/source/class/osparc/theme/Appearance.js (hunk 1 above)
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/theme/Appearance.js:147]: this should now be resolved

**Pull request merged**